account_advance_payment/__openerp__.py

```diff
@@ -25,7 +25,7 @@
 #
 {
     "name": "Account Advance Payment",
-    "version": "1.6",
+    "version": "9.0.0.1.6",
```

Comment:
why `1.6` is taken as minor and patch version when using as full module version `9.0.0.1.6` instead of taken it as mayor and minor versions, I mean `9.0.1.6.0`?